server/integration_tests/nl_screenshot_test.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import logging
+import multiprocessing
+import os
+import sys
+
+from flask_testing import LiveServerTestCase
+import requests
+
+from nl_server.__init__ import create_app as create_nl_app
+from server.__init__ import create_app as create_web_app
+import server.lib.config as libconfig
+import server.lib.util as libutil
+
+# Explicitly set multiprocessing start method to 'fork' so tests work with
+# python3.8+ on MacOS.
+# https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods
+# This code must only be run once per execution.
+# if sys.version_info >= (3, 8) and sys.platform == "darwin":
+#   multiprocessing.set_start_method("fork")
+#   os.environ['no_proxy'] = '*'
+
+_dir = os.path.dirname(os.path.abspath(__file__))
+
+_NL_SERVER_URL = 'http://127.0.0.1:6060'
```

Comment:
can use app.config['NL_ROOT']

Reply:
The code below explicitly sets 6060, so it perhaps makes sense to hardcode this too?

(this is copied from nl integration test)